lightkurve/targetpixelfile.py

```diff
@@ -1130,6 +1131,124 @@ def cutout(self, center=None, size=5):
         return self.__class__(newfits)
 
 
+    def inspect_pixels(self, ax=None, normalized=False, periodogram=False, aperture_mask=None, style='lightkurve', title=None, **kwargs):
```

Comment:
Nitpicking: let's name the argument `normalize` instead of `normalized`, for consistency with the `LightCurve.plot()` methods which also accept a `normalize` argument.
```suggestion
    def inspect_pixels(self, ax=None, normalize=False, periodogram=False, aperture_mask=None, style='lightkurve', title=None, **kwargs):
```